docs/sphinx-hoverxref.md

```diff
@@ -0,0 +1,28 @@
+# sphinx-hoverxref - preview cross-references
+
+Using sphinx-hoverxref, we can preview cross-references in the documentation.
+
+For instance, try placing your mouse over {hoverxref}`this reference to the front page <intro>`.
+We have used a named "target" for the reference. Here are some examples of how to name your targets:
+
+```{tab} reStructuredText
+
+```rest
+.. _foobar:
+
+My Headline
+===========
+
+Here is a paragraph, we link to this headline :hoverxref:`like this <foobar>`
+```
+
+```{tab} MyST
+
+```markdown
+
+ (foobar)=
+ 
+ # My Headline
+ 
+ Here is a paragraph, we link to this headline {hoverxref}`like this <foobar>`.
+```
```

Comment:
I modified the above code on my office workstation but forgot to push -- the most important change was that the MyST example comes first now.